File: totosuki/ABC/C/205.py
import sys
input = sys.stdin.buffer.readline

A, B, C = map(int, input().split())

if C % 2 == 0:
  A, B = map(abs, (A, B))
  if A > B: print(">")
  elif A < B: print("<")
  else: print("=")
elif C % 2 == 1:
  if A > B: print(">")
  elif A < B: print("<")
  else: print("=")

File: OCH_Tao/ABC/B/205.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const N = parseInt(input[0], 10);
let A = input[1].split(" ").map(x => parseInt(x, 10));
A.sort((a, b) => a - b);
for (let i = 0; i < N; i++) {
  if (A[i] !== i + 1) {
    console.log("No");
    return 0;
  }
}
console.log("Yes");